### DISCUSSION TITLE
Filter Pull Request should include all in created/assigned/mentioned/review requested

### DISCUSSION THREAD

**Discussion by notlaein** [community/community#17845] [Mobile] [answered]
When filtering Pull Request in iOS app I miss the filter to see all PRs in a repository. In case Im not creator/assigned/mentioned/review-requested. Please consider adding such a option to that filter.

**Comment by HenSquared** [comment, marked as answer]
Hey @einar-notland,

Thank you for this feature request. I've created an issue to track this and discuss with the team. While we don't have this behavior right from the Pull Requests on the Home tab, you can get a list of PRs for a certain repository by following below steps: 

1. Tap repositories on the home tab
2. Tap the repository you are interested in 
3. Tap pull requests

This will bring you to a list of all pull requests with options for filtering and ordering. 

Is this helpful?

**Reply by hellolennert** [reply]
Same as https://github.com/orgs/community/discussions/17856
